src/reseat/reseat.scss

```diff
@@ -0,0 +1,14 @@
+* {
+  box-sizing: border-box;
+}
+
+h2,
+ul,
+p,
+body,
+html {
+  font-family: "Roboto", sans-serif;
+  list-style: none;
+  padding: 0;
+  margin: 0;
```

Comment:
why do you set these styles for all tags?

p doesn't have list-style, body doesn't have paddings and so on